lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -169,22 +173,38 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
       M.select_install_rm_cmd(cache_folder, project_name..'-tmp')
     }
   else
+    local git_folder = utils.join_path(cache_folder, project_name)
+    local clone_error = 'Error during download, please verify your internet connection'
+    if is_windows then
+      clone_error = clone_error .. ". If on Windows you may need to enable Developer mode"
+    end
+
     return {
       {
         cmd = 'git',
         info = 'Downloading...',
-        err = 'Error during download, please verify your internet connection',
+        err = clone_error,
         opts = {
           args = {
             'clone',
             '--single-branch',
             '--branch', repo.branch or 'master',
-            '--depth', '1',
```

Comment:
depth 1 was used to only download the commit required, this will basically download a whole branch of commits ? what was the aim of removing that ?

Reply:
To make the pinning work. `--depth=1` only works for branches